src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write code here
+const listEl = document.querySelector('ul');
+
+const convertToNumber = str => Number(str.replace(/[^\d.-]/g, ''));
```

Comment:
This function `convertToNumber` is a good use of a regular expression to remove all non-numeric characters and convert the string to a number. However, it might be better to handle the case where the string can't be converted to a number. You could return `NaN` in such cases or throw an error.